lib/reindex/s3_bucketd.py

```diff
@@ -52,18 +53,24 @@ def inner(*args, **kwargs):
 MPU = namedtuple('MPU', ['bucket', 'key', 'upload_id'])
 BucketContents = namedtuple('BucketContents', ['bucket', 'obj_count', 'total_size'])
 
+class MaxRetriesReached(Exception):
+    def __init__(self, url):
+        super().__init__('Max retries reached for request to %s'%url)
+
 class BucketDClient:
 
     '''Performs Listing calls against bucketd'''
     __url_format = '{addr}/default/bucket/{bucket}'
     __headers = {"x-scal-request-uids": "utapi-reindex-list-buckets"}
 
-    def __init__(self, bucketd_addr=None):
+    def __init__(self, bucketd_addr=None, max_retries=5):
```

Comment:
It could be better to stay consistent with the script default and set 2 retries by default here, or even define this number of retries in a script global constant.